extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfigPersistenceUnit.java

```diff
@@ -329,9 +329,18 @@ public enum NullOrdering {
         @ConfigItem(defaultValue = "none")
         public NullOrdering defaultNullOrdering;
 
+        /**
+         * Enables IN clause parameter padding which improves statement caching.
+         *
```

Comment:
Can you also remove the extra line breaks?

Reply:
Just seeing this after I pushed, should I still make this change?

Reply:
Yes

Reply:
done.